app/javascript/components/public_health/Import.js

```diff
@@ -79,6 +81,18 @@ class Import extends React.Component {
         });
       })
       .catch(err => {
+        let validation_errors = [];
+        Object.keys(err.response.data).forEach(prop => {
+          validation_errors.push({
+            prop,
+            message: err.response.data[String(prop)],
+          });
+        });
+        const patientIndex = this.state.patients.findIndex(statePatient => _.isEqual(statePatient, patient.patient));
```

Comment:
Do we need to use the more expensive Lodash method here or can we just check the IDs?

Reply:
There are no ids assigned to these users because they are not imported yet so we really need to check the expensive way.